checkov/common/bridgecrew/platform_integration.py

```diff
@@ -127,7 +129,12 @@ def clean(self) -> None:
         self.platform_integration_configured = False
         self.http: urllib3.PoolManager | urllib3.ProxyManager | None = None
         self.http_timeout = urllib3.Timeout(connect=REQUEST_CONNECT_TIMEOUT, read=REQUEST_READ_TIMEOUT)
-        self.http_retry = urllib3.Retry(REQUEST_RETRIES, redirect=3)
+        self.http_retry = urllib3.Retry(
```

Comment:
it seems that there was already an auth retry mechanism and we narrowed the conditions for it, is that intended? (I mean we still do `REQUEST_RETRIES` retries at most which was the case before)

Reply:
POST methods and 401 are unhandled by default using urllib3. In this MR we include them.